cli/tests/pcluster/cli_commands/test_compute_fleet_status_manager.py

```diff
@@ -8,6 +10,10 @@
 class TestComputeFleetStatusManager:
     @pytest.fixture
     def compute_fleet_status_manager(self, mocker):
+        if "AWS_DEFAULT_REGION" not in os.environ:
+            # We need to provide a region to boto3 to avoid no region exception.
+            # Which region to provide is arbitrary.
+            os.environ["AWS_DEFAULT_REGION"] = "us-east-1"
```

Comment:
I'm not sure about this mocking, because in some tests there is: `boto3.client("dynamodb", region_name="us-east-1")`

so the region is not arbitrary. must be aligned with the region used in the test